nethttp/client.go

```diff
@@ -58,12 +59,22 @@ func ComponentName(componentName string) ClientOption {
 
 // ClientTrace returns a ClientOption that turns on or off
 // extra instrumentation via httptrace.WithClientTrace.
+// If this option is not used, the default behaviour is to
+// inject the span context.
```

Comment:
did you mean to add this comment to InjectSpanContext ?

Reply:
Good catch!